dicoogle/src/main/java/pt/ua/dicoogle/server/SOPList.java

```diff
@@ -218,7 +225,7 @@ public synchronized int updateTS(String UID, boolean [] p, boolean a) {
      * @return -1 if something went wrong, 1 otherwise
      */
     public synchronized int updateTSField(String UID, String name, boolean value) {
-        logger.debug("UID: {}, name: {}, value: {}", UID, name, value);
+        logger.info("UID: {}, name: {}, value: {}", UID, name, value);
```

Comment:
Given that this method is being called inside a 2-level loop, logging at info level is going to be quite spammy. Is this intentional?